spec/requests/api/service_dialogs_spec.rb

```diff
@@ -192,4 +192,137 @@ def init_dialog
       )
     end
   end
+
+  context 'Creates service dialogs' do
+    let(:dialog_request) do
+      {
+        :description => 'Dialog',
+        :label       => 'dialog_label',
+        :content     => {
+          :dialog_tabs => [
+            {
+              :description   => 'Dialog tab',
+              :position      => 0,
+              :label         => 'dialog_tab_label',
+              :dialog_groups => [
+                {
+                  :description   => 'Dialog group',
+                  :label         => 'group_label',
+                  :dialog_fields => [
+                    {
+                      :name  => 'A dialog field',
+                      :label => 'dialog_field_label'
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      }
+    end
+
+    it 'rejects service dialog creation without appropriate role' do
+      api_basic_authorize
+
+      run_post(service_dialogs_url, dialog_request)
+
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it 'rejects service dialog creation with an href specified' do
+      api_basic_authorize collection_action_identifier(:service_dialogs, :create)
+
+      run_post(service_dialogs_url, dialog_request.merge!("href" => service_dialogs_url(123)))
+      expected = {
+        "error" => a_hash_including(
+          "kind"    => "bad_request",
+          "message" => a_string_matching(/id or href should not be specified/)
+        )
+      }
+      expect(response.parsed_body).to include(expected)
+      expect(response).to have_http_status(:bad_request)
+    end
+
+    it 'rejects service dialog creation with an id specified' do
+      api_basic_authorize collection_action_identifier(:service_dialogs, :create)
+
+      run_post(service_dialogs_url, dialog_request.merge!("id" => 123))
+      expected = {
+        "error" => a_hash_including(
+          "kind"    => "bad_request",
+          "message" => a_string_matching(/id or href should not be specified/)
+        )
+      }
+      expect(response.parsed_body).to include(expected)
+      expect(response).to have_http_status(:bad_request)
+    end
+
+    it 'supports single service dialog creation' do
+      api_basic_authorize collection_action_identifier(:service_dialogs, :create)
+
+      expected = {
+        "results" => [
+          a_hash_including(
+            "description" => "Dialog",
+            "label"       => "dialog_label"
```

Comment:
@imtayadeway right now, yes. But i supposed it would make more sense to add it all in